src/pvecontrol/task.py

```diff
@@ -5,6 +5,8 @@
 from pvecontrol.node import NodeStatus
 
 
+columns = ["upid", "exitstatus", "node", "type", "starttime", "endtime", "runningstatus", "description"]
```

Comment:
"description" can be removed we slash it in [PR#15](https://github.com/enix/pvecontrol/pull/15)